src/components/CarList.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import '../styles/CarList.css';
+import { Link } from 'react-router-dom';
+import PlustIcon from '../assets/CarLIstImages/pluse.png';
+
+//금액 숫자 형식
+const formatCurrency = (value: number): string => {
+  return new Intl.NumberFormat('en-US', { style: 'decimal' }).format(value);
+};
+
+interface CarProductProps {
+  id: string;
+  titleimg: string;
+  productimg: string;
```

Comment:
단어의 조합인 경우 카멜케이스 작성법을 활용해 구분해주면 더 가독성이 좋아집니다~
```suggestion
  titleImg: string;
  productImg: string;
```